### PR TITLE
Feat/ce send

### PR DESCRIPTION
**Pull request by TheLeoB**
## Description (optional)
Move Exchange address to first on list, dark mode fix for empty state headers

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-info-components/src/Colors/DarkMode.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ const DarkTheme: DefaultTheme = {
   grey700: lighten(0.1, '#50596B'),
   grey800: lighten(0.1, '#B1B8C7'),
   grey900: '#121D33',
-  blue000: '#ECF5FE',
+  blue000: '#1c1c1c',
```
Comment (schnogz) [inline packages/blockchain-info-components/src/Colors/DarkMode.ts:27]: Intended?

File: packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const HeaderTextWrapper = styled.div`
 
 export const SceneHeaderText = styled(Text)`
   font-size: 32px;
-  color: black;
+  color: ${props => props.theme.textBlack};
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Layout/index.ts:63]: 👍

File: packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/selectors.js [hunk 3]
```diff
@@ -19,12 +17,10 @@ export const getData = createDeepEqualSelector(
         .map(path([0, 'state']))
         .map(complement(equals(TIERS_STATES.NONE)))
   ],
-  (userTiersR, userDataR, hasEmail, userCreatedR) => {
-    return lift((userTiers, userData, userCreated) => ({
-      userData,
-      userTiers,
+  (hasEmail, userCreatedR) => {
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/selectors.js:20]: Intended?

File: packages/blockchain-info-components/src/Colors/DarkMode.ts (hunk 1 above)
Comment (TheLeoB) [reply packages/blockchain-info-components/src/Colors/DarkMode.ts:27]: yeah, the lighter background looked odd in dark mode, so switched it to a darker color. checked other places that color was used, looked good too.

Comment (schnogz) [reply packages/blockchain-info-components/src/Colors/DarkMode.ts:27]: 👌

File: packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/selectors.js (hunk 3 above)
Comment (TheLeoB) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/selectors.js:20]: yes, tested. all good, thanks.

**Pull request merged**